UI/NativeApp.cpp

```diff
@@ -339,12 +342,13 @@ static void PostLoadConfig() {
 	else
 		i18nrepo.LoadIni(g_Config.sLanguageIni, langOverridePath);
 
-	g_threadManager.Init(cpu_info.num_cores, cpu_info.logical_cpu_count);
+#if PPSSPP_PLATFORM(ANDROID)
+	CreateDirectoriesAndroid();
+#endif
 }
 
-static bool CreateDirectoriesAndroid() {
-	// On Android, create a PSP directory tree in the external_dir,
-	// to hopefully reduce confusion a bit.
+bool CreateDirectoriesAndroid() {
+	// TODO: Really not sure why this code is Android-exclusive, except the ".nomedia" part.
```

Comment:
You added this for permissions iirc because it needed to happen after permissions were granted.

-[Unknown]

Reply:
Right..